index.html

```diff
@@ -439,12 +438,14 @@ <h1>About Nextcloud</h1>
 				<div class="footer-nav">
 					<h1>Resources</h1>
 					<ul>
+						<li><a class="reference external" href="https://nextcloud.com/search/">Search our website</a></li>
 						<li><a class="reference external" href="https://nextcloud.com/install">Download</a></li>
-						<li><a class="reference external" href="https://apps.nextcloud.com">App Store</a></li>
-						<li><a class="reference external" href="https://docs.nextcloud.com">Documentation</a></li>
-						<li><a class="reference external" href="https://nextcloud.com/enterprise">Enterprise Subscription</a></li>
+						<li><a class="reference external" href="https://docs.nextcloud.com/server/20/admin_manual/">Admin manual</a></li>
```

Comment:
See: https://github.com/nextcloud/nextcloud.com/blob/1d1f1fd5a09ac6828c5b706d23d673063cc1a7bd/strings.php#L15

Maybe we should also fix this over there but this does not seem to be there for no reason, since dev and user-manual is linking to `latest`.

Reply:
Maybe @jospoortvliet can tell us why this was done :)

Reply:
At some point, 'latest' was broken, but if it works, it's better I'd say ;-)

Sorry for the late reply.

Reply:
oh and I should then update it on the website, too...

Reply:
Done that.